src/actions/filesDropped/metadata.js

```diff
@@ -1,43 +1,53 @@
 import { rgb } from "d3-color";
 import { errorNotification, successNotification, warningNotification } from "../notifications";
 import { ADD_EXTRA_METADATA } from "../types";
-import { parseCsvTsv } from "./parseCsvTsv";
+import { parseCsv } from "./parseCsv";
 
 
 const handleMetadata = async (dispatch, getState, file) => {
   const fileName = file.name;
+  const reader = new FileReader();
+  reader.onload = async (event) => {
+    try {
+      const XLSX = (await import("xlsx/xlsx.mini")).default;
+      /* Convert tabular file to CSV string */
+      /* Only reads in the data from the first sheet of Excel files */
+      const workbook = XLSX.read(event.target.result, { type: 'binary' });
+      const firstSheet = workbook.Sheets[workbook.SheetNames[0]];
+      const sheetAsCsv = XLSX.utils.sheet_to_csv(firstSheet);
```

Comment:
Is there a need to go through CSV and then parse with `papaparse`? The XLSX library seems to correctly split my test CSV/TSVs into cells, and then we could use `XLSX.utils.sheet_to_json(firstSheet)`. Are there cases where special characters etc need this roundtrip - I would think these fail to be read correctly into cells initially.

Reply:
I thought about dropping `papaparse` but I couldn't find equivalent options in `xlsx` for `comments` and `dynamicTyping`. If these aren't important to keep, then we can drop `papaparse`.

Reply:
Probably worth keeping it for those benefits. It's only 7kb, and only fetched when this functionality is needed.